### PR TITLE
Sentiment Detector

### LINKED ISSUE #211: A script that will check sentiment for an input text without any external lib [GUI]
# Aim

- A script that will check sentiment for an input text using tkinter.
This can be helpful for people who use Social media a lot and would want to get how positive or negative their tweet,post,etc. is.
Usually positive text has more reach so it can be a "sentiment checker"

# Details

- I will implement a script that will take a text as an input via GUI and give the user how positive or negative their tweet/text is.
I will implement this using only python so no need for any other external library or dependency.
The only things the script will use are two text files "positive.txt" containing a list of positive words and a "negative.txt" containing a list of negative words.

On running the script a GUI in tkinter would open up and the user can paste their text and c.ick on a "get  score" nutton to find out of how positive or negative the input text is.

### LINKED ISSUE #949: Sentiment Detector GUI
**Is your feature request related to a problem? Please describe.**
I would like to add a  Sentiment Detector GUI

**Describe the solution you'd like**
I will use vaderSentiment and tkinter libraries

### PR DESCRIPTION
**Pull request by Amit366**
# Description

Added a script to detect sentiment of a sentence.

## Fixes #949 

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

yes

### CODE REVIEW

File: Script to check Sentiment/script.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
```
Comment (kaustubhgupta) [inline Script to check Sentiment/script.py:1]: wrong implementation. It's mentioned very clearly in the title also that you cannot use any external library for this: 

image

Kind;y check the issue description again and then make changes to this PR

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Script to check Sentiment/script.py (hunk 1 above)
Comment (Amit366) [reply Script to check Sentiment/script.py:1]: @kaustubhgupta then is there any way to find the percentage of positive, negative and neutral.

Comment (kaustubhgupta) [reply Script to check Sentiment/script.py:1]: I think you need to create a separate corpus for positive, negative, and neutral cases. This is also mentioned in the issue description

**Comment by kaustubhgupta** [comment]
updates on this?

**Comment by Amit366** [comment]
@kaustubhgupta I am not able to do a separate corpus

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta I am not able to do a separate corpus

Hmm okay, I have a solution. Create an issue named: "Sentiment detector GUI" and mention in the issue description that it will use these xyz libraries. I will assign that issue to you and you can update this PR description with that issue number. But importantly, that issue will be of level2. Are you okay with this? In this way, your contributions won't get vained. Let me know.

**Comment by Amit366** [comment]
@kaustubhgupta ok sir thank you

**Comment by Amit366** [comment]
@kaustubhgupta done

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]
@Amit366 💯

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**